packages/repo-collector/src/webapp/webapp.ts

```diff
@@ -116,6 +123,9 @@ function convertDatapoint(
               })),
           }
         : undefined,
+    sonarCloud: {
```

Comment:
nit: a suggestion that accompanies the suggestion about extending `WebappMetricDataRepoDatapoint` with a bit more information
```typescript
    sonarCloud: {
      enabled: !!datapoint.sonarCloud,
      testCoverage: datapoint.sonarCloud?.component?.measures?.find(el => el.metric === "coverage")?.value
    }
```